lib/tasks/data/supplier_locations.yml

```diff
@@ -319,7 +319,6 @@ suppliers:
     - MPS25
```

Comment:
maybe this is the right time to deprecate this file since its no longer in use

Reply:
yep, good call 👍